packages/ui/src/composites/wui-network-button/index.ts

```diff
@@ -21,7 +21,7 @@ export class WuiNetworkButton extends LitElement {
   // -- Render -------------------------------------------- //
   public override render() {
     return html`
-      <button data-testid="w3m-network-button" ?disabled=${this.disabled}>
+      <button data-testid="wui-network-button" ?disabled=${this.disabled}>
```

Comment:
Sane?

Reply:
Fixing the data test id since it's better to have `wui-` instaed of `w3m-`